dashboards/Experimental/MongoDB_Cluster_Summary_(new).json

```diff
@@ -3088,8 +2906,8 @@
         "allFormat": "glob",
         "current": {
           "selected": false,
-          "text": "mycluster2",
-          "value": "mycluster2"
+          "text": "test",
```

Comment:
Is this correct?

Reply:
Hi, this is dynamic and changes per instance.